src/t2iapi/metric/service.proto

```diff
@@ -133,4 +133,14 @@ service MetricService {
    */
   rpc DisplayMetricWithDifferentUnit (BasicHandleRequest)
       returns (BasicResponse);
+
+  /*
+  For the metric with the provided handle perform the following transition:
+   - provide pm:MetricValue/@Value resp. @Samples with pm:MetricQuality/@Validity = 'Vld' then
+   - change its accuracy such that it does not meet the limits specified either by an applicable standard that the
+     SDC METRIC PROVIDER demonstrates conformity to or in the ACCOMPANYING INFORMATION of the SDC METRIC PROVIDER
+   - provide a new pm:MetricValue/@Value resp. @Samples.
```

Comment:
i wopuld also add "and then" to be sure that the new MetricValue is provided while the last instruction still applies

```suggestion
   - change its accuracy such that it does not meet the limits specified either by an applicable standard that the
     SDC METRIC PROVIDER demonstrates conformity to or in the ACCOMPANYING INFORMATION of the SDC METRIC PROVIDER and then
   - provide a new pm:MetricValue/@Value resp. @Samples.
```